builtins/amp-ad.js

```diff
@@ -265,15 +247,10 @@ export function installAd(win) {
      * @private
      */
     sendAdIntersection_() {
```

Comment:
sending message is iframe/embed specfic stuff - would it still go in base/custom - element?